### PR TITLE
Sphinx docs

### PR DESCRIPTION
**Pull request by dakotabenjamin**
My first attempt, it's a start!

### CODE REVIEW

**Comment by danbjoseph** [comment]
what about putting the docs in their own repository? keep things a little cleaner and you could have some sort of travis-ci or circl-ci build process for the docs site?

File: doc/source/conf.py [hunk 1]
```diff
@@ -0,0 +1,178 @@
...
+project = u'OpenDroneMap'
+copyright = u'2017, OpenDroneMap'
+author = u'OpenDroneMap'
+
+# The version info for the project you're documenting, acts as replacement for
+# |version| and |release|, also used in various other places throughout the
+# built documents.
+#
+# The short X.Y version.
+version = u'0.4'
```
Comment (pierotofy) [inline doc/source/conf.py:61]: Should we read this from https://github.com/OpenDroneMap/OpenDroneMap/blob/master/VERSION ?

Comment (dakotabenjamin) [reply doc/source/conf.py:61]: Yes!

**Comment by dakotabenjamin** [comment]
@danbjoseph I was originally thinking to possibly use readthedocs.io as a hosting solution. Would this be possible with your suggestion?

**Comment by danbjoseph** [comment]
i was originally going to suggest using travis-ci to build the static pages from the master branch and push it to a gh-pages. but seems like readthedocs would probably be easier. would be happy to help set either up.

**Comment by danbjoseph** [comment]
although they do show ads on your site
https://docs.readthedocs.io/en/latest/ethical-advertising.html

**Comment by dakotabenjamin** [comment]
True, that is a concern. @smathermather maybe you could provide input? We may not be able to use that service in that case.

**Comment by danbjoseph** [comment]
i set up this site https://github.com/IFRCGo/ecv-toolkit to use travis-ci to build and push to gh-pages branch. i could do the same thing with sphinx.

**Comment by dakotabenjamin** [comment]
I created a odm-docs repo and added you, if you wanna have a go at it.

**Comment by danbjoseph** [comment]
awesome. will do so this weekend.

**Comment by danbjoseph** [comment]
good to go: https://github.com/OpenDroneMap/odm-docs

**Comment by smathermather** [comment]
We proceeded with travis-ci and not readthedocs, right?

**Comment by danbjoseph** [comment]
we're using a sphinx theme by readthedocs, but creating the static pages using the sphinx-build code as part of a travis-ci process and pushing to gh-pages.
so no ads or anything and the only reference to readthedocs is a link at the bottom of the page.
https://opendronemap.github.io/odm-docs/
screen shot 2018-03-14 at 4 29 01 pm

**Comment by smathermather** [comment]
Ah, that's fine then. Thanks @danbjoseph.

**Comment by dakotabenjamin** [comment]
Thanks Dan!

**Pull request closed without merging**